builder/dockerfile/metrics.go

```diff
@@ -0,0 +1,31 @@
+package dockerfile
+
+import "github.com/docker/go-metrics"
+
+var (
+	buildsTriggered        metrics.Counter
+	buildsTriggeredFailed  metrics.LabeledCounter
+	buildsTriggeredSuccess metrics.Counter
+)
+
+func init() {
+	buildMetrics := metrics.NewNamespace("builder", "dockerfile", nil)
+
+	buildsTriggered = buildMetrics.NewCounter("builds_triggered", "Number of triggered image builds")
+	buildsTriggeredFailed = buildMetrics.NewLabeledCounter("builds_triggered_failed", "Number of failed image builds", "reason")
```

Comment:
@alfred-landrum just to initialize the `metric` with all possible scenarios. It's recommended to do that. /cc @brian-brazil